client/verta/verta/_registry/modelversions.py

```diff
@@ -29,5 +29,7 @@ def _call_back_end(self, msg):
     def _create_element(self, msg):
         return RegisteredModelVersion(self._conn, self._conf, msg)
 
-    def with_model(self, registered_model=None):
-        raise NotImplementedError
+    def with_model(self, registered_model):
```

Comment:
Let's keep the `registered_model=None` in the parameters.
```suggestion
    def with_model(self, registered_model=None):
```
This will be useful to reset the object to not use a model.

Reply:
done